### ISSUE TITLE
The order of a message's item is strange

### ISSUE THREAD

**Issue by sukiletxe** [UnigramDev/Unigram#2880]
I cannot remember when this was changed, but now when navigating a message using object navigation I get the following order:

* Unlabeled link, which takes me to the sender's profile
* Message text (or whatever content and controls)
* Message time
* Sender's nickname
* Sender's admin title (if any)
* And lastly, the message being replied to (if any).

I think that the original message should go right before or right after the current message, and the time should go last. I think the previous ordering went like this, and it was more logical than the current one.

Windows 10 Versión 21H2 (compilación de SO 19044.2130), Unigram 9.0 (7840) X64 Direct

**Comment by FrayxRulez** [comment]
The order of the elements affects the way they're rendered in the UI, and this is why this was done (even if now I can't remember exactly why this was done). Unfortunately there's no practical way to change this at the moment...

**Comment by FrayxRulez** [comment]
As there was no follow ups I consider this closed

**Comment by sukiletxe** [comment]
By now I got used to this order, although I still consider it strange. Muscle memory I guess. Since the last message was yours and you didn't ask for any information I found no reason to follow up.
